src/regional_settings.h

```diff
@@ -22,7 +22,7 @@ class building_bin
         weighted_int_list<overmap_special_id> buildings;
         std::map<overmap_special_id, int> unfinalized_buildings;
     public:
-        building_bin() {};
+        building_bin() {}
```

Comment:
You can default those constructors: `building_bin() = default;` - similar in other places.

Reply:
Thanks. I've did that were applicable.

Reply:
Hmm.. It seems clang doesn't like this.

Reply:
@narc0tiq: What's clang version on Jenkins PR builder? Is it 3.8?

Reply:
Should've been in the build output, but I think I ballsed that up at some revision of the mainline build script. But yes:
```
clang version 3.8.0-2ubuntu4 (tags/RELEASE_380/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

Reply:
The clang error relates specifically to the way in which `selection_column_preset` is used; the others should probably be fine.  (It also looks like a clang-3.8 bug).

Reply:
We'll see now.

Reply:
`overmap_terrain_special` was also used in the same way, so I had to update its constructor too.